lib/teleterm/api/protogen/golang/v1/service.pb.go

```diff
@@ -252,7 +251,7 @@ type CredentialInfo struct {
 	sizeCache     protoimpl.SizeCache
 	unknownFields protoimpl.UnknownFields
 
-	Username string `protobuf:"bytes,1,opt,name=Username,proto3" json:"Username,omitempty"`
```

Comment:
This decapitalization won't cause any problems, right? Just checking

Reply:
It looks like it has no practical effects on the generated files, both the Go and JavaScript versions.

Reply:
Yep, I think it's safe.